src/mlpack/methods/reinforcement_learning/replay/random_replay.hpp

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file random_replay.hpp
+ * @author Shangtong Zhang
+ *
+ * This file is an implementation of random experience replay.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_RL_REPLAY_RANDOM_REPLAY_HPP
+#define MLPACK_METHODS_RL_REPLAY_RANDOM_REPLAY_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace rl {
+
+/**
+ * Implementation of random experience replay.
+ *
+ * At each time step, interactions between the agent and the
+ * environment will be saved to a memory buffer. When necessary,
+ * we can simply sample previous experiences from the buffer to
+ * train the agent. Typically this would be a random sample and
+ * the memory will be a First-In-First-Out buffer.
+ *
+ * For more information, see the following.
+ *
+ * @code
+ * @phdthesis {lin1993reinforcement,
```

Comment:
Really minor issue: the build tool doesn't tolerate a whitespace between the command and the bracket: ``@phdthesis{lin1993reinforcement``